modular-docs-manual/content/topics/creating_procedure_modules.adoc

```diff
@@ -18,29 +27,37 @@ The introduction typically provides context for the procedure, such as:
 * Why and where the user performs the procedure
 * Special considerations specific to the procedure
 
-.Procedure prerequisites
-This section is optional. Prerequisites are a bulleted list of conditions that must be satisfied before the user starts the procedure. Use a bulleted list for prerequisites and the plural heading *Prerequisites*, even if you only have a single prerequisite.
+.Limitations
+This section is optional and not used often. If there are any limitations that the affect the procedure, list them in this section.
+
+.Prerequisites
+This section is optional. Prerequisites are a bulleted list of conditions that must be satisfied before the user starts the procedure. Use a bulleted list for prerequisites and the plural heading *Prerequisites*, even if your procedure only has a single prerequisite.
 
 Prerequisites can be full sentences or sentence fragments; however, prerequisite list items must be parallel.
 
 Focus on relevant prerequisites that users might not otherwise be aware of. Do not list obvious prerequisites. If a prerequisite applies to all of the procedures in a user story, consider listing the prerequisite in the assembly file. If you do this, consider including the prerequisite in the procedure module as a comment.
 
-.Procedure steps
-The procedure consists of one or more steps required to complete the procedure. Each step describes one action. For single-step procedures, use an unnumbered bullet instead of a numbered list.
+.Procedure
+This section is required. The procedure consists of one or more steps required to complete the procedure. Each step describes one action written in the imperative form, for example, 'Do this action'. For single-step procedures, use an unnumbered bullet instead of a numbered list.
 
 NOTE: Not all numbered lists in documentation are procedures. You can also use numbered lists in any module type for non-procedural sequences, such as a process flow of system actions.
 
-.Procedure verification
+.Verification or Verification steps
```

Comment:
do we need 'steps'?

Reply:
+1 agree that Verification on its own would be good.

Reply:
+1 for me too
Do we also need to mention that it is not recommended to preface a step with the word Verification, but instead add this step to the Verification section instead, to promote consistency across the documentation set?